src/v1/util.ts

```diff
@@ -0,0 +1,45 @@
+import * as crypto from "crypto-js";
```

Comment:
This UTIL can be outside of Versions, because generateSignature or most of the URLs will not change
In future if anything get changed only will to those versions code

Reply:
The code is designed as Modular Design Pattern. So the dependencies should not kept outside or being shared.

For V2 we create a new folder and wrap everything. So we can break the dependency from v1 and up coming versions.

Reply:
Atleast move  generic methods like generate_signatures I don't think signature generation and all will change